dspace-api/src/main/java/org/dspace/mock/MetadataValueDTO.java

```diff
@@ -18,17 +19,20 @@
  *
  * @author kevinvandevelde at atmire.com
  */
-public class MockMetadataValue {
+
+
+//TODO COMMENTS
```

Comment:
This TODO should be removed.  We might want to improve the JavaDocs on this class though to make it clear that this `MetadataValueDTO` is no longer just for crosswalks.   It's simply an "in-memory data transfer object (DTO) for the MetadataValue class".